change/@ni-nimble-tokens-53d62bd1-be2a-4fe1-9e78-13c91edb04b2.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Lint disable without behavior change",
+  "packageName": "@ni/nimble-tokens",
+  "email": "rajsite@users.noreply.github.com",
+  "dependentChangeType": "none"
+}
```

Comment:
GitHub's rich diff for the package-lock.json (click the 📄 icon) shows changes to nimble dependencies. Is that normal? 

![image](https://github.com/ni/nimble/assets/10500124/df9dc1d3-cb8f-4428-9710-0fe3bca6b9a6)

Reply:
Discussed offline and these are normal diffs for a PR that modifies package versions.